[GC-4976] Use wp_enqueue commands

alexoakland opened the pull request:
### GC-4976
Link to ticket: https://bynder.atlassian.net/browse/GC-4976

### What has been done
- Moved styling and javascript into dedicated files and enqueued them

alexoakland committed:
Moved styling and javascript into dedicated files and enqueued them

alexoakland commented on assets/css/system-info.css:
```diff
@@ -0,0 +1,11 @@
+#system-info-textarea {
```
@Gallmond @CWDN would like to get your thoughts on this approach. A dedicated CSS and JS file when required.

Idea being that it's easy to find the related resources seeing as my IDE does not link anything! (That might just be my IDE not being setup for Wordpress more than anything)

I'm not married to the idea so happy to go towards a general `custom.css` files a `custom.js` or something like that if people prefer!

CWDN replied:
I could be wrong but I think we tend to add to the `sass` within `assets/css/sass/gathercontent-importer.scss` then it will get compiled and pulled in automatically. Saves enqueuing more files.

Think you can do this with the JS as well and get grunt to compile the JS for you, it will at least be minified and all the other stuff required for Wordpress.

alexoakland committed:
Allow CSS and JS files to be enqueued onto all pages

Also moved redirect and system info related styles and JS into the
grunt generated files.

alexoakland committed:
Added the system-info and redirect JS to main JS file

alexoakland committed:
The files updated by grunt build
